tools/upload_text.php

```diff
@@ -488,4 +515,24 @@ function ensure_path_is_unused( $path )
     }
 }
 
+function delete_contents($dir)
```

Comment:
To make this less dangerous, consider moving it into the Project class as `delete_smoothreading_dir()`. Then validate that the project's directory, and then smooth directory exists. That both limits how this function will be used (to prevent future programmer error) and moves the context closer to the object it is working on (the Project).